src/__mocks__/utils.js

```diff
@@ -1,4 +1,4 @@
-const actualUtils = require.requireActual('../utils')
+const actualUtils = jest.requireActual('../utils')
```

Comment:
Was getting eslint errors for this. `require.requireActual` is deprecated in favor of `jest.requireActual`